src/EditorFeatures/CSharpTest2/app.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <appSettings>
+    <add key="xunit.useAppDomain" value="true"/>
+    <add key="xunit.diagnosticMessages" value="false"/>
+    <add key="xunit.parallelizeTestCollections" value="true"/>
+  </appSettings>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
+      <dependentAssembly>
+        <assemblyIdentity name="System.Collections.Immutable" publicKeyToken="b03f5f7f11d50a3a" culture="neutral"/>
+        <bindingRedirect oldVersion="1.0.0.0-1.65535.65535.65535" newVersion="1.1.37.0"/>
```

Comment:
I just copied the same file from CSharpTest.  So if this is wrong then htey're both wrong.